virtualcluster/pkg/syncer/syncer.go

```diff
@@ -233,25 +233,6 @@ func (s *Syncer) Run(stopChan <-chan struct{}) {
 		}
 	}()
 	go wait.Until(s.healthPatrol, 1*time.Minute, stopChan)
-	go func() {
```

Comment:
@m-messiah  Why do you remove this go func?

Reply:
@Fei-Guo thanks for catching this up. I made a revert PR for this part of code: https://github.com/kubernetes-sigs/cluster-api-provider-nested/pull/288 

I retried it and it looks like when I hit `remove redundant return` in Goland - it removes this goroutine too..

Reply:
Thanks for catching that Fei!